index.html

```diff
@@ -0,0 +1,163 @@
+            <div class="middle">
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fas fa-external-link-square-alt"></i> Show Live</a
+              ><br />
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fab fa-github-square"></i> GitHub</a
+              >
+            </div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+          </section>
+          <section class="project">
+            <h3>Project 4</h3>
+            <img src="images/pexels-digital-buggu-374559.jpg" alt="Project 4" />
+            <div class="middle">
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fas fa-external-link-square-alt"></i> Show Live</a
+              ><br />
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fab fa-github-square"></i> GitHub</a
+              >
+            </div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+          </section>
+          <section class="project">
+            <h3>Project 5</h3>
+            <img src="images/pexels-digital-buggu-374559.jpg" alt="Project 5" />
+            <div class="middle">
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fas fa-external-link-square-alt"></i> Show Live</a
+              ><br />
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fab fa-github-square"></i> GitHub</a
+              >
+            </div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+          </section>
+          <section class="project">
+            <h3>Project 6</h3>
+            <img src="images/pexels-digital-buggu-374559.jpg" alt="Project 6" />
+            <div class="middle">
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fas fa-external-link-square-alt"></i> Show Live</a
+              ><br />
+              <a href="#" target="_blank" class="project-btn"
+                ><i class="fab fa-github-square"></i> GitHub</a
+              >
+            </div>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+          </section>
+        </div>
+        <div class="back-to-top">
+          <a href="#top-nav" title="Back to Top" alt="Back to Top"
+            ><i class="far fa-arrow-alt-circle-up nav-icon"></i
+          ></a>
+        </div>
+      </div>
+      <div class="profile-page" id="about-me">
+        <h2>About Me</h2>
+        <div class="profile">
+          <section class="profile-col">
+            <img
+              src="images/pexels-the-world-hopper-1851481.jpg"
+              title="Traveller photographer at the pyramids"
+              alt="Traveller photographer at the pyramids"
+            />
+          </section>
+          <section class="profile-col">
+            <p>
```

Comment:
Contents of `<p>` on same line (consistency)

Reply:
Fixed in formatting settings